org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
+      <!-- Temperature scale:  'f' for Fahrenheit. 'c' for celsius -->
+      <arg name="scale" type="s" direction="in" />
```

Comment:
I like the sound of 'Unit' better too, but this is definitely going to break the current translators. The Wink thermostat is currently in a pull request, and it will be merged soon. I think one good practice is to make any schema changes in a separate PR because they can potentially break existing code. 

Personally I feel like Celsius (for now) should be the preferred unit and then we don't worry about error handling and parameters and we make it clear in the documentation. The conversion I feel should be handled client-side.